### ISSUE TITLE
The directory is not under git source control

### ISSUE THREAD

**Issue by victor-mesquita** [ForkIssues/Tracker#969]
Hi @DanPristupov 

I'm trying to open an existing repository but fork show me this message:

![image](https://user-images.githubusercontent.com/5200285/79050041-c4fbcc80-7bfd-11ea-9e8e-2d2e537658df.png)

its works when i open the repository in terminal and run git status

![image](https://user-images.githubusercontent.com/5200285/79050090-10ae7600-7bfe-11ea-8aed-b4eb15a78395.png)

**Comment by DanPristupov** [comment]
Hi.
Does log file contain any warning?
`~/Library/Logs/Fork.log`

Could you show outputs of the following commands please?

```
git rev-parse --show-toplevel
git rev-parse --absolute-git-dir
```

**Comment by victor-mesquita** [comment]
**Fork.log:** 
ℹ️> Fork Version: 1.0.92 Build: 1.0.92 PID: 89168
ℹ️> XCGLogger Version: 7.0.1 - Level: Debug
ℹ️> XCGLogger writing log to: file:///Users/victormesquita/Library/Logs/Fork.log
ℹ️[AppDelegate.swift:111] applicationDidFinishLaunching(_:) > Git instance: system
ℹ️[RemoteServiceNotificationManager.swift:73] stop() > Stop checking GitHub notifications
ℹ️[RemoteServiceNotificationManager.swift:48] start() > Start checking GitHub notifications
ℹ️[RemoteServiceNotificationManager.swift:95] refresh(force:) > Refresh GitHub notifications...
ℹ️[RemoteServiceNotificationManager.swift:95] refresh(force:) > Refresh GitHub notifications...

**git rev-parse --show-toplevel output:**
/Users/victormesquita/Documents/Projects/Beta/points

**git rev-parse --absolute-git-dir output:**
/Users/victormesquita/Documents/Projects/Beta/points/.git

edit:

I changed the git version to 2.18.0 and its works now

![image](https://user-images.githubusercontent.com/5200285/79050911-76513100-7c03-11ea-927e-07b704e1488e.png)

**Comment by DanPristupov** [comment]
> git 2.18.0

22 Jun **2018**

This is a 4 years old git. Could you try to switch Fork to the internal git instance?

**Comment by victor-mesquita** [comment]
> > git 2.18.0
> 
> 22 Jun **2018**
> 
> This is a 4 years old git. Could you try to switch Fork to the internal git instance?

The bug apparently is gone. 
thanks!

**Comment by DanPristupov** [comment]
If I remember correctly, Fork wasn't publicly released yet when git 2.18 came out :).

**Comment by TheMates** [comment]
Late for the party, but got the same error - "The directory is not under source control".
I was switching between users and turns out git had a problem with that. When under a different user than created or cloned the repo for the first time, you need to call this, to make it available for other user.

`git config --global --add safe.directory /path_to_the/repository`

Then Fork loaded it successfully.
